dev/local/install.sh

```diff
@@ -1,9 +1,26 @@
 #!/bin/bash
+# Bootstrap delphi-epidata
+#
+# Downloads the repos needed for local delphi-epidata development into current dir 
+# and provides a Makefile with Docker control commands.
 
-mkdir -p driver/repos/delphi driver-logs/delphi_database_epidata driver-logs/delphi_web_epidata
-cd driver/repos/delphi
+mkdir -p repos/delphi
```

Comment:
I still recommend building into a `driver` directory. Reasoning:

1. scripts developed in local dev will be more easily adapted to run on staging or prod, where this directory is required
2. Dangling a bare Makefile in a random directory that may include other files is risky practice

We can however put the logs directory into driver, which will be more tidy.

Reply:
My (unstated) assumption was that the user is already in a clean directory like `../delphi-epidata-dev/` when they call this script. I agree now that it's not a good assumption to make.

My main pickiness is that the directory tree already seems unnecessarily deep:
```
delphi-epidata-dev
└── driver
    └── repos
        └── delphi
            └── delphi-epidata
```
and the `delphi` level is where finally get to any code. So it's really about just saving one level of depth for me
```
delphi-epidata-dev
└── repos
    └── delphi
        └── delphi-epidata
```
If it were up to me, I would get rid of the `delphi` directory in there too, since it's not doing much anymore (especially so when we get rid of the legacy `undefx` tools). 

But yea, it's just a minor aesthetic thing. And if prod requires `driver`, then it makes sense to make local dev reflect that. So I'll bring `driver` back and get rid of `docker-logs`.